### DISCUSSION TITLE
Helm chart

### DISCUSSION THREAD

**Discussion by doron-cohen** [riverqueue/riverui#90] [Ideas]
We would love that 🙂

**Comment by bgentry** [comment]
I’m not sure I understand what you’re asking for. We already publish Docker images which can easily be run in Kubernetes. What could we put in a shared public helm chart that isn’t super specific to an individual Kubernetes deployment?

**Comment by virtualroot** [comment]
I made a Helm chart with the official Docker image.

```
$ helm repo add virtualroot https://charts.virtualroot.dev
$ helm install riverui --set databaseURL="postgresql://user:pass@localhost/river" virtualroot/riverui 
```

@doron-cohen I hope it helps with your use case.

**Reply by brandur** [reply]
Thanks for this! @doron-cohen neither Blake or I are Helm users, so would suggest using this one for the time being. If we see more people asking for it, we can formalize and add it to the docs.

**Reply by doron-cohen** [reply]
Great. Thanks @virtualroot!
